src/hooks/getters/useUser.ts

```diff
@@ -8,14 +8,17 @@ const useUser = (
 	user: UserWithAccount | null
 	loading: boolean
 	error: boolean
-	refetch: () => void
+	refetch: (shouldChangeLoadingState?: boolean) => void
```

Comment:
It has a default value set to `true`, thus keeping compatibility with existing code (cause this is normal behavior for loading users, I believe). Only in a single case, when we re-fetching user after NFT creation it passed as `false` in order to prevent full page blinking/reload and showing success modal state. So I believe this one should be kept "as is" (unless you have a better idea of re-fetching user content when we create NFT)